### ISSUE TITLE
Could not find load file STKDoc_testsConfig.cmake

### ISSUE THREAD

**Issue by bavier** [trilinos/Trilinos#218]
With Trilinos 12.6.1 configured with STK, I get the following error when using the installed cmake configuration fragments:

```
CMake Error at /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/STK/STKConfig.cmake:143 (INCLUDE):
  include could not find load file:

    /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/STK/../STKDoc_tests/STKDoc_testsConfig.cmake
Call Stack (most recent call first):
  /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/TrilinosCouplings/TrilinosCouplingsConfig.cmake:146 (INCLUDE)
  /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/Trilinos/TrilinosConfig.cmake:146 (INCLUDE)
  CMakeLists.txt:66 (FIND_PACKAGE)


CMake Error at /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/TrilinosCouplings/TrilinosCouplingsConfig.cmake:147 (INCLUDE):
  include could not find load file:

    /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/TrilinosCouplings/../STKDoc_tests/STKDoc_testsConfig.cmake
Call Stack (most recent call first):
  /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/Trilinos/TrilinosConfig.cmake:146 (INCLUDE)
  CMakeLists.txt:66 (FIND_PACKAGE)


CMake Warning at /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/Trilinos/TrilinosConfig.cmake:156 (MESSAGE):
  Component "STKDoc_tests" NOT found.
Call Stack (most recent call first):
  CMakeLists.txt:66 (FIND_PACKAGE)


CMake Error at /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/Trilinos/TrilinosConfig.cmake:203 (include):
  include could not find load file:

    /tmp/bavier/trilinos-12.6.1-cce-x86/lib/cmake/Trilinos/../STKDoc_tests/STKDoc_testsConfig.cmake
Call Stack (most recent call first):
  CMakeLists.txt:66 (FIND_PACKAGE)
```

STKConfig.cmake lists `STKDoc_tests` in `STK_PACKAGE_LIST` but there is no corresponding STKDoc_testsConfig.cmake.

This patch appears to fix the issue:

```
--- trilinos-12.6.1-Source/packages/stk/stk_doc_tests/CMakeLists.txt    2016-01-12 13:17:58.000000000 -0600
+++ trilinos-12.6.1-Source/packages/stk/stk_doc_tests/CMakeLists.txt    2016-03-16 12:12:46.000118000 -0500
@@ -1,3 +1,4 @@
+TRIBITS_SUBPACKAGE(Doc_tests)

 TRIBITS_ADD_TEST_DIRECTORIES(stk_mesh)

@@ -7,3 +8,5 @@

 TRIBITS_ADD_TEST_DIRECTORIES(stk_io)
 TRIBITS_ADD_TEST_DIRECTORIES(stk_util)
+
+TRIBITS_SUBPACKAGE_POSTPROCESS()
```

**Comment by bartlettroscoe** [comment]
We will catch and eliminate all problems like this once we snapshot the TriBITS changes in TriBITSPub/TriBITS#232.  Hopefully we will not find too may cases like this in Trilinos.

**Comment by bartlettroscoe** [comment]
Looks like this got fixed a long time ago in the commit 6bbc751.

Closing as complete.
